airbyte-integrations/connectors/source-surveymonkey/integration_tests/expected_records.txt

```diff
@@ -0,0 +1,427 @@
+{"stream":"survey_responses","data":{"id":"12731350053","recipient_id":"","collection_mode":"default","response_status":"completed","custom_value":"","first_name":"","last_name":"","email_address":"","ip_address":"91.242.192.88","logic_path":{},"metadata":{"contact":{}},"page_path":[],"collector_id":"405843688","survey_id":"307785408","custom_variables":{},"edit_url":"https://www.surveymonkey.com/r/?sm=uQGI1ouCt3TwYg_2F0OjcapOOokxhfAvoP3vsYcJntZXz4yNaoZRXNQePI1o27JEfS","analyze_url":"https://www.surveymonkey.com/analyze/browse/BPAkhAawaMN8C17tmmNFxvvr7YEM4YZ5j7dMDUtBHEw_3D?respondent_id=12731350053","total_time":32,"date_modified":"2021-06-10T11:02:07+00:00","date_created":"2021-06-10T11:01:34+00:00","href":"https://api.surveymonkey.com/v3/surveys/307785408/responses/12731350053","pages":[{"id":"168831381","questions":[]},{"id":"168831382","questions":[{"id":"667461606","answers":[{"choice_id":"4385174252"}]},{"id":"667461628","answers":[{"choice_id":"4385174361"}]},{"id":"667461630","answers":[{"choice_id":"4385174370"}]},{"id":"667461651","answers":[{"choice_id":"4385174483"}]},{"id":"667461652","answers":[{"choice_id":"4385174496"}]}]},{"id":"168831388","questions":[{"id":"667461666","answers":[{"choice_id":"4385174599"}]},{"id":"667461670","answers":[{"choice_id":"4385174621"}]},{"id":"667461674","answers":[{"choice_id":"4385174634"}]},{"id":"667461676","answers":[{"choice_id":"4385174646"}]},{"id":"667461686","answers":[{"choice_id":"4385174678"}]}]}]},"emitted_at":1674149668082}
```

Comment:
This IP address is in here a lot `91.242.192.88` - I assume it's from us? It geolocates to Boryspil, Kyivska, Ukraine

Reply:
as for IPs: I suppose they belong to internet provers and usually dynamically change. So as for me they should not be a problem as well.